docs/topics/forms/formsets.txt

```diff
@@ -448,6 +448,47 @@ happen when the user changes these values::
     {'pub_date': datetime.date(2008, 5, 11), 'ORDER': 1, 'title': 'Article #2'}
     {'pub_date': datetime.date(2008, 5, 10), 'ORDER': 2, 'title': 'Article #1'}
 
+``ordering_widget``
+-------------------
+
+.. attribute:: BaseFormSet.ordering_widget
+
+Default: :class:`~django.forms.NumberInput`
+
+Lets you change the order field widget::
+
+    class FormSet(BaseFormSet):
+        ordering_widget = HiddenInput
+
+    class ArticleForm(ModelForm):
+        class Meta:
+            model = Article
+            fields = ('content', )
```

Comment:
To be consistent with other examples in this page, I would use a simpler example without creating a custom form but with all required imports (like in `can_order` doc), e.g.
```
    >>> from django.forms import BaseFormSet, formset_factory
    >>> from myapp.forms import ArticleForm
    >>> class BaseArticleFormSet(BaseFormSet):
    ...     ordering_widget = HiddenInput

    >>> ArticleFormSet = formset_factory(ArticleForm, formset=BaseArticleFormSet, can_order=True)
```